dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -491,6 +490,25 @@ public static boolean isNumeric(String str) {
         return true;
     }
 
+    public static boolean isNumber(String str) {

```

Comment:
How about something like this:

```java
public static boolean isNumeric(String str, boolean allowDot) {
        if (str == null) {
            return false;
        }
        int sz = str.length();
        for (int i = 0; i < sz; i++) {
            if ((!allowDot && str.charAt(i) == '.') || !Character.isDigit(str.charAt(i))) {
                return false;
            }
        }
        return true;
    }
```

Not tested yet. Just for your information. In this way we can unify the logic into one method.
And call isNumeric(value, true) from ClassHelper.

Reply:
I have modified it.